### ISSUE TITLE
Keyboard zoom is broken for the scatterplot viewer

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#4316]
## Package
lib-classifier

## Describe the bug
I can zoom in and out with the +/- keys, but when I pan right and left with the arrow keys, the chart scale is reset to 1. 

## To Reproduce
1. Go to https://www.zooniverse.org/projects/hughdickinson/superwasp-black-hole-hunters/classify/workflow/17386
2. Zoom in with + or the zoom in button.
3. Pan right and left with the arrow keys.

## Expected behavior
The chart should pan right and left while staying zoomed-in.
